lib/collection/header-list.js

```diff
@@ -30,11 +30,22 @@ _.assign(HeaderList.prototype, /** @lends HeaderList.prototype */ {
      * @returns {Number}
      */
     contentSize: function () {
-        if (!this.count()) { return 0; }
+        var raw = E,
+
+            headers = this.reduce(function (acc, header) {
+                // unparse header only if it has a valid key and is not disabled
+                if (header && header.key && !header.disabled) {
```

Comment:
Let’s not extract header.key here. I know that size will be a bit off - but for really edge cases and we don’t want to bleed the node issue all over our codebase.